lib/ansible/plugins/doc_fragments/files.py

```diff
@@ -33,7 +33,7 @@ class ModuleDocFragment(object):
     type: raw
   owner:
     description:
-    - Name of the user that should own the filesystem object, as would be fed to I(chown).
+    - Name of the user or gid that should own the filesystem object, as would be fed to I(chown).
```

Comment:
This part of the change is incorrect; you cannot pass a GID to `owner`.

Reply:
Sorry. I changed the wrong line. It should be ok now.